frontend/src/concepts/analyticsTracking/trackingProperties.ts

```diff
@@ -3,7 +3,10 @@ export type ODHSegmentKey = {
 };
 
 export type IdentifyEventProperties = {
+  isAdmin: boolean;
   anonymousID?: string;
+  userId?: string;
```

Comment:
`userId` isn't used anywhere. Should this be removed? I also thought that we shouldn't include identifiable information in events.

Reply:
It could indeed be removed from _this_ PR.  UserId would be the web-user-id for the sandbox case.